client/src/worldgen.rs

```diff
@@ -155,10 +155,89 @@ pub fn voxels(graph: &mut DualGraph, node: NodeId, cube: Vertex) -> VoxelData {
                 }
             }
         }
+
+        // Planting trees on dirt. Trees consist of a block of wood and a block of leaves.
+        // The leaf block is on the opposite face of the wood block as the dirt block.
+        let loc: usize = 2;
+        let voxel_of_interest_index =
+            index(absolute_subchunk_coords(loc, loc, loc, subchunk_offset));
+        let neighbor_data = voxel_neighbors(
+            na::Vector3::new(loc, loc, loc),
+            subchunk_offset,
+            &mut voxels,
+        );
```

Comment:
It looks like this hardcodes the only possible tree location as voxel coordinates 2,2,2. Is that intentional?

Reply:
Yes, I did intend to hardcode them like that.

Reply:
Consider `na::Vector3::repeat(loc)` here.